OsmAnd/src/net/osmand/plus/wikivoyage/data/TravelGpx.java

```diff
@@ -77,9 +77,13 @@ public GpxTrackAnalysis getAnalysis() {
 	@Override
 	public WptPt createWptPt(@NonNull Amenity amenity, @Nullable String lang) {
 		WptPt wptPt = new WptPt();
+		wptPt.setName(amenity.getName());
 		wptPt.setLat(amenity.getLocation().getLatitude());
 		wptPt.setLon(amenity.getLocation().getLongitude());
-		wptPt.setName(amenity.getName());
+
+		Map<String, String> wptPtExtensions = wptPt.getExtensionsToWrite();
+		amenity.getNamesMap(true).forEach((key, value) -> wptPtExtensions.put("name:" + key, value));
```

Comment:
Do we need to check also here for "ref" && "sym"?

Reply:
The tags name:ref and name:sym are synthetic and used only for Ways (not for WptPt).